src/lengths.jl

```diff
@@ -17,4 +23,4 @@ function gyroradius(B, T::EnergyOrTemp; mass_numb=1, Z=1)
     return gyroradius(B, Vperp; mass_numb, Z)
 end
 
-gyroradius(val, B::BField) = gyroradius(B, val)
\ No newline at end of file
+gyroradius(val::Union{Velocity, EnergyOrTemp}, B::BField; kw...) = gyroradius(B, val; kw...)
```

Comment:
We should be consistent about the order of arguments, or we should offer keyword arguments that are not order dependent.